spec/quickbooks-payments/charge_spec.rb

```diff
@@ -3,6 +3,19 @@
 RSpec.describe Quickbooks::Payments::Charge do
   include_context 'setup access_token'
 
+  let(:valid_attributes) do
+    {
+      created: '2014-11-03T16:41:42Z',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.